packages/preview3-shim/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "@bytecodealliance/preview3-shim",
+  "version": "0.1.0",
+  "description": "WASI Preview3 shim for JS environments",
+  "author": "Guy Bedford, Eduardo Rodrigues<16357187+eduardomourar@users.noreply.github.com, Tomasz Andrzejak>",
+  "type": "module",
+  "types": "./types/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./types/index.d.ts",
+      "node": "./lib/nodejs/index.js"
+    },
+    "./*": {
+      "types": "./types/*.d.ts",
+      "node": "./lib/nodejs/*.js"
+    }
+  },
+  "scripts": {
+    "test": "vitest --run",
+    "bench": "vitest bench --run"
+  },
+  "files": [
+    "types",
+    "lib"
+  ],
+  "dependencies": {
+    "@bytecodealliance/preview2-shim": "file:../preview2-shim"
```

```suggestion
    "@bytecodealliance/preview2-shim": "file:../preview2-shim"
```
We won't be able to publish with this reference (or actually what happens is you publish a package that breaks after download) -- could you depend on `@bytecodealliance/preview2-shim` here?